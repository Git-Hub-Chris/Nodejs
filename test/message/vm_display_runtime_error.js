'use strict';
const common = require('../common');
const assert = require('assert');
const vm = require('vm');

console.error('beginning');

vm.runInThisContext('throw new Error("boo!")', { filename: 'test.vm' });

console.error('end');
