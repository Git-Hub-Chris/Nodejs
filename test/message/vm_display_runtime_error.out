beginning
test.vm:1
throw new Error("boo!")
^

Error: boo!
    at test.vm:1:7
    at Script.runInThisContext (node:vm:*)
    at Object.runInThisContext (node:vm:*)
    at Object.<anonymous> (*test*message*vm_display_runtime_error.js:*)
    at ModuleCompile (node:internal/modules/cjs/loader:*)
    at Module._compile (node:internal/modules/cjs/loader:*)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:*)
    at Module.load (node:internal/modules/cjs/loader:*)
    at Function.Module._load (node:internal/modules/cjs/loader:*)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:*)
test.vm:1
throw new Error("spooky!")
^

Error: spooky!
    at test.vm:1:7
    at Script.runInThisContext (node:vm:*)
    at Object.runInThisContext (node:vm:*)
    at Object.<anonymous> (*test*message*vm_display_runtime_error.js:*)
    at ModuleCompile (node:internal/modules/cjs/loader:*)
    at Module._compile (node:internal/modules/cjs/loader:*)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:*)
    at Module.load (node:internal/modules/cjs/loader:*)
    at Function.Module._load (node:internal/modules/cjs/loader:*)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:*)
